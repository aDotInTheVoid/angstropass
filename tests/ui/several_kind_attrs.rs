#[angstropass::langs]
mod langs {
    mod l1 {
        #[add]
        #[replace]
        type Foo = i32;
    }
}

fn main() {}
