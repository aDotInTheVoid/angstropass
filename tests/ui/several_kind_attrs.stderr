error: only one of #[add], #[replace], or #[delete] is allowed
 --> tests/ui/several_kind_attrs.rs:5:9
  |
5 |         #[replace]
  |         ^^^^^^^^^^
